### DISCUSSION TITLE
EddieHub Interactive session planning and suggestions

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [EddieHubCommunity/support#1629] [Show and tell]
|Name     |         Description  |              Relative to GMT     |        Emoji |
| :---:|:---: |:---: |:---: |
GMT      |    Greenwich Mean Time   |    GMT              |             :smile: |
UTC  |  Universal Coordinated Time    | GMT               |            :tada: | 
ECT   |          European Central Time    |  GMT+1:00    |            :joy: | 
EET    |          Eastern European Time   |  GMT+2:00     |          :thumbsup: | 
ART   | (Arabic) Egypt Standard Time  | GMT+2:00      |        :raised_hands: | 
EAT     |       Eastern African Time        |   GMT+3:00     |        :calendar:  | 
MET     |         Middle East Time          |     GMT+3:30    |          :compass: | 
NET      |         Near East Time                |   GMT+4:00    |          :cry:   | 
PLT      |       Pakistan Lahore Time         |  GMT+5:00     |        :white_check_mark: | 
IST      |        India Standard Time          |   GMT+5:30     |         :gift: | 
BST     |     Bangladesh Standard Time |   GMT+6:00      |       :rocket: | 
VST    |      Vietnam Standard Time        | GMT+7:00      |        :pencil: | 
CTT     |       China Taiwan Time             |  GMT+8:00       |       :hammer: | 
JST      |       Japan Standard Time         |    GMT+9:00      |       :hammer_and_pick:  | 
ACT    |      Australia Central Time        |    GMT+9:30        |    :computer: | 
AET    |       Australia Eastern Time          | GMT+10:00     |     :mag:  | 
SST     |       Solomon Standard Time     |    GMT+11:00     |       :astonished: | 
NST    |    New Zealand Standard Time |    GMT+12:00      |     :1st_place_medal:  |  
MIT     |     Midway Islands Time         |     GMT-11:00     |          :fire: | 
HST    |      Hawaii Standard Time         |   GMT-10:00       |      :sweat_smile: |  
AST     |        Alaska Standard Time       |    GMT-9:00        |     :brain: | 
PST     |     Pacific Standard Time          |       GMT-8:00      |      :mechanical_arm: | 
PNT    |     Phoenix Standard Time       |      GMT-7:00       |      :iphone:  | 
MST    |    Mountain Standard Time     |     GMT-7:00        |      :wine_glass:  | 
CST     |    Central Standard Time         |      GMT-6:00       |      :cloud:  | 
EST     |         Eastern Standard Time         |   GMT-5:00     |       :1234:  | 
IET       |     Indiana Eastern Standard Time  | GMT-5:00    |        :100:  | 
PRT     |    Puerto Rico & US Virgin Islands Time  |  GMT-4:00 |  :-1:  | 
CNT     |    Canada Newfoundland Time |       GMT-3:30   |       :smiling_imp: |  
AGT     |     Argentina Standard Time |       GMT-3:00       |      :gem: | 
BET      |     Brazil Eastern Time        |             GMT-3:00      |      :up: | 
CAT     |     Central African Time     |               GMT-1:00      |     :sunglasses: | 

***If you have any different timezones then let us know. Also if any above timezone has only 30  mins or 1hr difference then consider it as your timezone please.***

### You can react as an comment. If your timezone emoji is available (Someone already commented it) then click the upward arrow to that answer

**Comment by ghost** [comment]
🎁

**Comment by Rahma-AlWadhahi** [comment]
🥲

**Reply by kaiwalyakoparkar** [reply]
This is :cry:

**Comment by imvisp** [comment]
what have you desided ?

**Reply by kaiwalyakoparkar** [reply]
Waiting yet! Many timezone people are not up now!
